src/commands/solved.ts

```diff
@@ -10,7 +10,7 @@ const command: Command = {
 		const embed = getDefaultEmbed()
 			.setTitle('If your issue is resolved, please consider doing the following:')
```

Comment:
```suggestion
			.setTitle('If your issue is resolved, please help by doing the following two steps:')
```
What do you think of this?